finance/management/commands/xformnetfilerawdata.py

```diff
@@ -113,7 +113,7 @@ def parse_benefactor(row, verbosity=1):
 
     elif row['entity_Cd'] == 'OTH':  # Commerial benefactor or Other
         benefactor, _ = models.OtherBenefactor.objects \
-            .get_or_create(name=clean_name(row.get('tran_NamL', '')))
+            .get_or_create(name=clean_name(row.get('tran_NamL')) or '')
```

Comment:
These lines should be equivalent, `get` takes a the default value as the second argument in case the key does not exist.